bubblewrap.c

```diff
@@ -1070,23 +1071,25 @@ privileged_op (int         privileged_op_socket,
       break;
 
     case PRIV_SEP_OP_REMOUNT_RO_NO_RECURSIVE:
-      bind_result = bind_mount (proc_fd, NULL, arg2, BIND_READONLY);
+      bind_result = bind_mount (proc_fd, NULL, arg2, BIND_READONLY, &failing_path);
 
       if (bind_result != BIND_MOUNT_SUCCESS)
-        die_with_bind_result (bind_result, errno,
+        die_with_bind_result (bind_result, errno, failing_path,
                               "Can't remount readonly on %s", arg2);
 
+      assert (failing_path == NULL);    /* otherwise we would have died */
```

Comment:
I find it a bit odd that we have this assertion here. It's like checking that `bind_mount()` does what it was supposed to.

Reply:
This is just a nitpick that was probably worth mentioning. Definitely not an issue to leave it as is.

Reply:
The idea is that it's an assertion that we aren't leaking memory here.